From 14a04408392fd593ba82de9df14a042c96e0ecf9 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 25 Nov 2014 13:44:55 -0500
Subject: [redhat] kernel.spec: add Module.kabi_ppc64le file as source

Message-id: <5fa6a711111d83b72f1e51fe6e24e3e7e99c269b.1416921247.git.poros@redhat.com>
Patchwork-id: 100877
O-Subject: [PATCH RHEL7.1 3/3] [redhat] kernel.spec: add Module.kabi_ppc64le file as source
Bugzilla: 1149269
RH-Acked-by: Kyle McMartin <kmcmarti@redhat.com>

Insert Module.kabi_ppc64le file as source.
    Reorder Module.kabi_{arch} files (cosmetic change)

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/.gitignore b/redhat/.gitignore
index 15e6464..6ea87b3 100644
--- a/redhat/.gitignore
+++ b/redhat/.gitignore
@@ -3,6 +3,7 @@ configs/kernel*.config
 configs/*-merged
 
 kabi/Module.kabi_ppc64
+kabi/Module.kabi_ppc64le
 kabi/Module.kabi_s390x
 kabi/Module.kabi_x86_64
 kabi/kabi-current
diff --git a/redhat/git/files b/redhat/git/files
index 53000f2..e155d85 100644
--- a/redhat/git/files
+++ b/redhat/git/files
@@ -14,6 +14,7 @@ cpupower.config
 cpupower.service
 sign-modules
 Module.kabi_ppc64
+Module.kabi_ppc64le
 Module.kabi_s390x
 Module.kabi_x86_64
 kernel-3.10.0-ppc64.config
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 2d34581..1232439 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -347,9 +347,10 @@ Source18: check-kabi
 
 Source20: Module.kabi_x86_64
 Source21: Module.kabi_ppc64
-Source22: Module.kabi_s390x
+Source22: Module.kabi_ppc64le
+Source23: Module.kabi_s390x
 
-Source23: kernel-abi-whitelists.tar.bz2
+Source25: kernel-abi-whitelists.tar.bz2
 
 Source50: kernel-%{version}-x86_64.config
 Source51: kernel-%{version}-x86_64-debug.config
@@ -1172,7 +1173,7 @@ INSTALL_KABI_PATH=$RPM_BUILD_ROOT/lib/modules/
 mkdir -p $INSTALL_KABI_PATH
 
 # install kabi releases directories
-tar xjvf %{SOURCE23} -C $INSTALL_KABI_PATH
+tar xjvf %{SOURCE25} -C $INSTALL_KABI_PATH
 %endif  # with_kernel_abi_whitelists
 
 %if %{with_perf}
-- 
1.7.1